Gemfile

```diff
@@ -25,6 +25,8 @@ group :development, :test do
   gem 'simplecov'
   gem 'shoulda-matchers'
   gem 'table_print', '~> 1.5', '>= 1.5.7'
+  gem "factory_bot_rails"
+  gem 'faker'
```

Comment:
Curious to what this is

Reply:
Faker is used a lot of times with factory_bot to auto generate randomized data with large variations. Faker is nice because your objects don't all look the same, and can seem more like data that would exist on a live app.

Reply:
I love faker and factory bot.